qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -1,26 +1,50 @@
 #!/bin/sh -ex
 #
-# rbd_mirror_journal.sh - test rbd-mirror daemon in journal-based mirroring mode
+# rbd_mirror.sh - test rbd-mirror daemon in snapshot and journal-based mirroring mode
+#
+# Usage:
+#   rbd_mirror.sh
+#
+# Use environment variable RBD_MIRROR_MODE to set the mode
+# Available modes: snapshot | journal
 #
 # The scripts starts two ("local" and "remote") clusters using mstart.sh script,
 # creates a temporary directory, used for cluster configs, daemon logs, admin
 # socket, temporary files, and launches rbd-mirror daemon.
 #
 
-. $(dirname $0)/rbd_mirror_helpers.sh
+if [ "${#}" -gt 0 ]; then
+    echo "unnecessary arguments: ${@}"
+    exit 100
+fi
 
+if [ -z "${RBD_MIRROR_MODE}" ]; then
+    echo "env variable RBD_MIRROR_MODE is empty, use it to set the mode, available modes: snapshot | journal"
+    exit 100
+elif [ "${RBD_MIRROR_MODE}" != "snapshot" ] && [ "${RBD_MIRROR_MODE}" != "journal" ]; then
+    echo "unknown mode: ${RBD_MIRROR_MODE}"
+    echo "hint: use env variable RBD_MIRROR_MODE to set the right mode, available modes: snapshot | journal"
```

Comment:
In this case the user would have already set RBD_MIRROR_MODE, just to something invalid, so no need to mention it again.  I'd condense the whole thing to a single branch:
```
if [ "${RBD_MIRROR_MODE}" != "snapshot" ] && [ "${RBD_MIRROR_MODE}" != "journal" ]; then
  echo "unknown mode: ${RBD_MIRROR_MODE}"
  echo "set RBD_MIRROR_MODE env variable, available modes: snapshot | journal"
  exit 100
fi
```